sdk/java/pulumi/src/main/java/com/pulumi/experimental/automation/OperationType.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2025, Pulumi Corporation
+
+package com.pulumi.experimental.automation;
+
+public enum OperationType {
+    UNKNOWN(-1),
+    SAME(0),
+    CREATE(1),
+    UPDATE(2),
+    DELETE(3),
+    REPLACE(4),
+    CREATE_REPLACEMENT(5),
+    DELETE_REPLACED(6),
+    READ(7),
+    READ_REPLACEMENT(8),
+    REFRESH(9),
+    READ_DISCARD(10),
+    DISCARD_REPLACED(11),
+    REMOVE_PENDING_REPLACE(12),
+    IMPORT(13),
+    IMPORT_REPLACEMENT(14);
+
+    private final int value;
```

Comment:
Do you need the `value`? I can't see anywhere it's used (maybe it's coming in a future PR, or I've missed it like an idiot), and I don't think you need it if all you care about is e.g. comparing the enum values.